### DISCUSSION TITLE
DAI 2023-2024 - My first IntelliJ + Maven project - @theodrosrun

### DISCUSSION THREAD

**Discussion by Theodrosrun** [heig-vd-dai-course/heig-vd-dai-course#48] [Show and tell]
https://github.com/Theodrosrun/dai-lab04

**Comment by ludelafo** [comment]
Thank you!

The actual README is empty; I would fill it to say a few words on what the project is about and how to build and run the app.

Feel free to fill the poll in the GitHub Discussion regarding this chapter: https://github.com/orgs/heig-vd-dai-course/discussions/3.

You can do the same with the chapter Git, GitHub and Markdown: https://github.com/orgs/heig-vd-dai-course/discussions/2.
